components/stakeForm/stakeForm.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+  FC,
+  useCallback,
+  useState,
+  EventHandler,
+  SyntheticEvent,
+  FormEvent,
+} from 'react';
+import { trackEvent } from '@lidofinance/analytics-matomo';
+import { Block, Input, Steth, Button } from '@lidofinance/lido-ui';
+
+import { MATOMO_CLICK_EVENTS } from 'config';
+import TxStageModal, { TX_STAGE } from 'components/txStageModal';
+
+import FutureTxInfo from './components/futureTxInfo';
+import { InputWrapper, PseudoFormWrapper } from './styles';
+import { stakeProcessing } from './utils';
+
+const submitTrackEvent = () => {
+  // PAY ATTENTION: Remove. Example just to showing how to use a matomo events tracking
+  // More details: https://github.com/lidofinance/warehouse/tree/main/packages/analytics/matomo
+  trackEvent(...MATOMO_CLICK_EVENTS.submitStake);
+};
+
+const StakeForm: FC = () => {
+  // Modals
+  const [isTxModalOpen, setIsTxModalOpen] = useState<boolean>(false);
+
+  const openTxModal = useCallback(() => {
+    setIsTxModalOpen(true);
+  }, []);
+
+  const closeTxModal = useCallback(() => {
+    setIsTxModalOpen(false);
+  }, []);
+
+  const [txStage, setTxStage] = useState(TX_STAGE.IDLE);
+  // /Modals
```

Comment:
Empty content after comment

Reply:
// /Something - closes the comment block for the code

// Start comment block
```code```
// /closes the comment block